datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -508,20 +514,19 @@ CREATE TABLE dcsa_im_v3_0.document_type (
 DROP TABLE IF EXISTS dcsa_im_v3_0.transport_event_type CASCADE;
 CREATE TABLE dcsa_im_v3_0.transport_event_type (
     transport_event_type_code varchar(4) PRIMARY KEY,
-    transport_event_type_name varchar(100) NOT NULL,
+    transport_event_type_name varchar(30) NOT NULL,
     transport_event_type_description varchar(250) NULL
 );
 
 DROP TABLE IF EXISTS dcsa_im_v3_0.empty_indicator CASCADE;
 CREATE TABLE dcsa_im_v3_0.empty_indicator (
-    empty_indicator_code varchar(5) PRIMARY KEY,
-    empty_indicator_name varchar(100) NOT NULL
+    empty_indicator_code varchar(5) PRIMARY KEY
 );
 
 DROP TABLE IF EXISTS dcsa_im_v3_0.event CASCADE;
 CREATE TABLE dcsa_im_v3_0.event (
     event_id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
-    event_type text NOT NULL,
+--  event_type text NOT NULL,
```

Comment:
```suggestion
```

Reply:
Do we need this? as we are aware at the time of fetching the data which type of event it is.

Reply:
I meant drop the line entirely rather than leaving it commented out.